doc(release) Release notes collect 25.10 avr.26 (#5224)
centreon/centreon-documentation: versioned_docs/version-25.10/releases/centreon-os.mdx

| **Centreon Open Tickets** | [25.10.5 (March 30, 2026)](#centreon-open-tickets-25105)<br/>[25.10.4 (February 25, 2026)](#centreon-open-tickets-25104)<br/>[25.10.3 (February 23, 2026)](#centreon-open-tickets-25103)<br/>[25.10.2 (January 29, 2026)](#centreon-open-tickets-25102)<br/>[25.10.1 (December 17, 2025)](#centreon-open-tickets-25101)<br/>[25.10.0 (November 4, 2025)](#centreon-open-tickets-25100) |
| **Centreon AWIE** | [25.10.5 (Febuary 25, 2026)](#centreon-awie-25105)<br/>[25.10.3 (January 26, 2026)](#centreon-awie-25103)<br/>[25.10.2 (December 31, 2025)](#centreon-awie-25102)<br/>[25.10.1 (December 17, 2025)](#centreon-awie-25101)<br/>[25.10.0 (November 4, 2025)](#centreon-awie-25100) |
| **Centreon DSM** | [25.10.2 (February 25, 2026)](#centreon-dsm-25102)<br/>[25.10.1 (December 17, 2025)](#centreon-dsm-25101)<br/>[25.10.0 (November 4, 2025)](#centreon-dsm-25100) |

</details>

## April 28, 2026

### Centreon Collect 25.10.5

<details open>
  <summary>Enhancements</summary>

> - Broker is now able to handle secured TLS connection.

Reviewer: ```suggestion
- Broker is now able to handle secure TLS connections.
```

> - [CMA] Added a feature to get public CA using fingerprint & establish TLS connection.

Reviewer: ```suggestion
- [CMA] The CMA agent can now retrieve the certificate automatically generated by the poller.
```

</details>

<details>
  <summary>Bug fixes</summary>

> - Fixed perl connector issue when plugins output is longer than 4096 bytes.

Reviewer: ```suggestion
- Fixed an issue occurring with the perl connector when plugin output was longer than 4096 bytes.
```

</details>

### Centreon Gorgone 25.10.6

<details>
  <summary>Bug fixes</summary>

> - [Config] Prevented application crashes when a service has no associated category. These services are now automatically assigned to a "NoCategory" dimension, as for hosts.

Reviewer: ```suggestion
- [Config] Fixed an issue that caused application crashes when a service had no associated category. These services are now automatically assigned to a "NoCategory" dimension, just like hosts.
```

> - [Packaging] Do not install centreon-common package when installing centreon-gorgone.

Reviewer: ```suggestion
- [Packaging] The **centreon-common** package is no longer installed when installing **centreon-gorgone**.
```

</details>

### Centreon Monitoring Agent 25.10.5

<details open>
  <summary>Enhancements</summary>

> - [CMA] Added a feature to get public CA using fingerprint & establish TLS connection.

Reviewer: ```suggestion
- [CMA] The CMA agent can now retrieve the certificate automatically generated by the poller.
```
